### DISCUSSION TITLE
[ FF7 ] Steam 7thHeaven2.0 startup error

### DISCUSSION THREAD

**Discussion by SurSmooth** [julianxhokaxhiu/FFNx#184] [General]
![ff7error](https://user-images.githubusercontent.com/72694231/95678723-56370600-0b83-11eb-8731-8f65d2ab1b65.png)
[FFNx.log](https://github.com/julianxhokaxhiu/FFNx/files/5361057/FFNx.log)

I fucked up and accidently clicked the crash log, now it tries to open in notepad and that is wrong app, and won't upload to here. SORRY!!!!

**Comment by julianxhokaxhiu** [comment]
Hi, thanks for having created the issue but as you're using 7th you must approach the relative author on Discord or Qhimm for support. The crash may be just related to some mods not behaving correctly :) Cheers
